data/desktop/meson.build

```diff
@@ -0,0 +1,15 @@
+untranslated_desktop_file = configure_file(
+  input: 'org.gnome.GPaste.Ui.desktop.in.in',
+  configuration: conf,
+  output: 'org.gnome.GPaste.Ui.desktop.in',
```

Comment:
Any way to compute the output name from the input name?

Reply:
Not without becoming way less simple. Same applies as above, simplicity > being smart.